internal/integration/integration_test.go

```diff
@@ -73,10 +74,17 @@ func TestEndToEnd(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	b := make([]byte, 1)
```

Comment:
Can you use https://github.com/google/exposure-notifications-server/blob/main/pkg/util/generate.go#L47 here instead? I'm trying to centralize random number generate as much as possible.